quickstarts/microsoft.datafactory/data-factory-v2-blob-to-blob-copy/main.bicep

```diff
@@ -0,0 +1,142 @@
+@description('Data Factory Name')
+param dataFactoryName string = 'datafactory${uniqueString(resourceGroup().id)}'
+
+@description('Location of the data factory. Currently, only East US, East US 2, and West Europe are supported.')
+param location string = resourceGroup().location
```

Comment:
if locations are restricted, remove the default and add allowedValues - though looking at the latest list I don't think that restriction is accurate.